nadya/first/task2.py

```diff
@@ -0,0 +1,10 @@
+"""
+задан рядок
+вивести кількість чисел у рядку
+"""
+text="ghkdlh5673bjhdvfl"
+count=0
+for i in text:
+    if i.isdigit():
+        count=count+1
+print(count)
```

Comment:
ok is everything
`def task2(text):
    count = 0
    for i in text:
        if i.isdigit():
            count = count + 1
    print(count)`